Gemfile

```diff
@@ -1,8 +1,29 @@
-source "http://rubygems.org"
+source 'https://rubygems.org'
 
-gem "rake"
-gem "puppet", ENV['PUPPET_VERSION'] || '~> 3.6.0'
-gem "puppet-lint"
-gem "puppet-syntax"
-gem "puppetlabs_spec_helper"
+group :development, :test do
+  gem 'rake', '~> 10.1.0',            :require => false
+  gem 'rspec-puppet',                 :require => false
+  gem 'puppetlabs_spec_helper',       :require => false
+  gem 'beaker-puppet_install_helper', :require => false
+  gem 'serverspec',                   :require => false
+  gem 'puppet-lint',                  :require => false
+  gem 'pry',                          :require => false
+  gem 'simplecov',                    :require => false
+  gem 'beaker',                       :require => false
```

Comment:
It's generally recomended to put the acceptance test gems into a seperate Gem group called "system_tests" so that travis doesn't have to install them when doing spec tests.